### DISCUSSION TITLE
neovim: editor integration using LSP/null-ls

### DISCUSSION THREAD

**Discussion by lbodor** [numtide/treefmt#236] [Show and tell]
Hi,

https://github.com/jose-elias-alvarez/null-ls.nvim has recently added basic support for `treefmt`. I use it with some extra config as shown in https://github.com/jose-elias-alvarez/null-ls.nvim/pull/1512.

**Comment by jfly** [comment]
As a slight variant of the above, I've added the `nix_flake_fmt` builtin to none-ls: https://github.com/nvimtools/none-ls.nvim/blob/90e4a27ccaa25979a6b732b9f06dfa43b54957b7/doc/BUILTINS.md#nix_flake_fmt. If you're using flakes, this is a nice way to configure your editor which doesn't even assume you're using treefmt.
